BeleenMergeShop Script (needs review) & ArtixWedding Storyscript

PapaShokry opened the pull request:
- BeleenMergeShop requires  number of member, seasonal, rare and special offer items so i cannot fully test it
Requested by:@D-A+R=2#1920
- ArtixWedding story script simple one just needed it for the merge shop

allenc9727 committed:
Fix YnR armor check

allenc9727 committed:
Update YnR to support all BM/SM variants

ShaunH321 committed:
Update Yokai.cs

PapaShokry committed:
BeleenMergeShop Script (needs review) & ArtixWedding Storyscript

- BeleenMergeShop requires  number of member, seasonal, rare and special offer items so i cannot fully test it
Requested by:@D-A+R=2#1920
- ArtixWedding story script simple one just needed it for the merge shop

PUNK3DAF committed:
CoreAwe: Member check fix

ShaunH321 committed:
Update WarFuryEmblem.cs

ShaunH321 committed:
Merge branch 'Skua' of https://github.com/BrenoHenrike/Scripts into Skua

PapaShokry committed:
BeleenMerge redundant fixes

ShaunH321 committed:
toInv: false for LoO and ShadowScythe

ShaunH321 committed:
Banking on Metal quests

ShaunH321 committed:
Remove bank non AC item

ShaunH321 committed:
Update CoreDailies.cs

ShaunH321 committed:
Update CoreDailies.cs

ShaunH321 committed:
Merge pull request #157 from allenc97/bots

Update YNR

allenc9727 committed:
Fix OBON name change

ShaunH321 committed:
Merge pull request #159 from allenc97/bots

Fix OBON name change

ShaunH321 committed:
yes and no added.

LordExelot committed:
Adv/StartBuyAllMerge: Now has itemBlackList

LordExelot committed:
Released BankAllItems

LordExelot committed:
Adv/BuyItem: Now buys the right quantity of a recursive buy

LordExelot committed:
Adv/canBuy: Now checks Achievements and Requiered-Items

ShaunH321 committed:
Create WorldCoreMerge.cs

LordExelot committed:
BankAllItems: Fixed

LordExelot committed:
Merge branch 'Skua' of https://github.com/BrenoHenrike/Scripts into Skua

LordExelot committed:
Core/SetOptions: Now has a listerner to help in the war against moderators

LordExelot committed:
Core/PrisonDetector: Added "Oh fuck!" button

LordExelot committed:
Core/PrisonDetector: Prevented multiple triggers

LordExelot committed:
Adv: Fixed typo in logger

PUNK3DAF committed:
CoreSoW2: New quest added

PUNK3DAF committed:
Merge branch 'Skua' of https://github.com/BrenoHenrike/Scripts into Skua

ShaunH321 committed:
Added option storage

PUNK3DAF committed:
HighSeasCommander[10y]: Quest ID fix

PUNK3DAF committed:
AstraviaCastleMerge: cs_include fix

LordExelot committed:
Fixed null

LordExelot committed:
Fucking autocorrect

PUNK3DAF committed:
AstraviaCastleMerge: Astravian Medal fix

ShaunH321 committed:
Create ArmySupplies.cs

wtffidy committed:
Update GoldFarm.cs

ShaunH321 committed:
fix

ShaunH321 committed:
Merge branch 'Skua' of https://github.com/BrenoHenrike/Scripts into Skua

wtffidy committed:
Hopefully the Uni35 ( Nulgathdemandswork )doesnt break :P tested it for awhile seemed fine.

wtffidy committed:
Release: (on 9/2/22) New Helm Enh

+ the wep Enh.. but they arent filled in yet.. stuff still runs tho :thumbsup:  (itll stop the script if & when it gets to the helm selected.

LordExelot committed:
MergeTemplateHelper: Removed redundant bit

LordExelot committed:
Core: Disabled Debuglogger

LordExelot committed:
Core/EnsureCompleteChoose temp fix

PapaShokry committed:
adding artixwedding to 0allstory as i forgot

LordExelot committed:
Core: Crashes now have semi-autoreport

LordExelot committed:
Core: Added yelling to EnsureLoad

LordExelot committed:
adv skills updated

wtffidy committed:
ight should be good to go

wtffidy committed:
Merge branch 'Skua' of https://github.com/BrenoHenrike/Scripts into Skua

wtffidy committed:
Update UnlockForgeEnhancements.cs

wtffidy committed:
Rbot Relic > converted to SKua

LordExelot committed:
Farms: Streamlined Blade of Awe REP

LordExelot committed:
Merge branch 'Skua' of https://github.com/BrenoHenrike/Scripts into Skua

wtffidy committed:
Release: ChronoMerge

Requested by: @D-A+R=2#1920

wtffidy committed:
Create StreamwarMerge.cs

LordExelot committed:
HBDK: Changed Venom head

LordExelot committed:
Merge branch 'Skua' of https://github.com/BrenoHenrike/Scripts into Skua

LordExelot committed:
Core: Added TryParse to inPublicRoom

LordExelot committed:
Farm: Added new CanChange

ShaunH321 committed:
VampireFix

ShaunH321 committed:
SukiFix possibly?!

Ghetto fix but it works so - huntmonsterid my ass.

ShaunH321 committed:
Fixes fixes fixes

wtffidy committed:
Update StreamwarMerge.cs

wtffidy committed:
Merge branch 'Skua' of https://github.com/BrenoHenrike/Scripts into Skua

ShaunH321 committed:
Removed EquipClass on storyline.

ShaunH321 committed:
Merge branch 'Skua' of https://github.com/BrenoHenrike/Scripts into Skua

tanigorlx committed:
Add files via upload

tanigorlx committed:
Changed lines 62-67 to equip solo class

tanigorlx committed:
Add files via upload

tanigorlx committed:
fix typo

tanigorlx committed:
fixed

tanigorlx committed:
Update FireandIceMerge.cs

wtffidy commented on Prototypes/BeleensMerge.cs:
```diff
@@ -0,0 +1,497 @@
+    public LegionExercise4 LegionExercise4 = new();
+    public CoreNation CoreNation = new();
+    public TarosManslayer TarosManslayer = new();
+    public ArtixWedding ArtixWedding = new();
```
if the 2nd part of the public is not used within the script, you do not need the public only the //cs_include for that specific script. please check the rest of your publics above here

wtffidy commented on Prototypes/BeleensMerge.cs:
```diff
@@ -0,0 +1,497 @@
+                    Core.EquipClass(ClassType.Farm);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.HuntMonster("Stalagbite", "Vath", req.Name, isTemp: false);
```
vath can be a bit wonky there is a bit in Corebots > line 1256: "Core.KillVath":
`public void KillVath(string? item = null, int quant = 1, bool isTemp = false, bool log = true, bool publicRoom = false)`
please use that instead 👍

wtffidy commented on Prototypes/BeleensMerge.cs:
```diff
@@ -0,0 +1,497 @@
+                    Core.FarmingLogger($"{req.Name}", quant);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.Logger("This item is not setup yet"); //needs to be edited later as i couldn't find ShopID for it
```
shopid:
`[1966]    Beleen's Pretty Pink Gear`

wtffidy commented on Prototypes/BeleensMerge.cs:
```diff
@@ -0,0 +1,497 @@
+                    }
+                    break;
+
+                case "Taro's Manslayer":
```
already exists ^_^ please use the pre-made function in: Nation\Various\TarosManslayer.cs
`public void GuardianTaro(bool ManslayerOnly = false)`

`TMS.GuardianTaro(true);`

--i added a non-mem bit for it that kills the dood for it

wtffidy commented on Prototypes/BeleensMerge.cs:
```diff
@@ -0,0 +1,497 @@
+                #endregion
+
+                case "Platinum Wings":
+                    if (!Core.IsMember)
```
for things that require a membership you could just wrap them in a
if (Core.IsMember)
{
stuff
}
🤷  if the player isn't a mem it'll just skip it 👏 -- check the rest as well please

wtffidy commented on Story/ArtixWedding.cs:
```diff
@@ -0,0 +1,107 @@
+        Story.KillQuest(3234, "GrimskullAnnex", "Grim Soldier");
+
+    }
+    public void ArtixWeddingQuests()
```
+1 line for prettyness 😆 above the public here or don't il do it later this file is fine either way

wtffidy reviewed (commented):
👍  boop me when this is done and it can be merged.

wtffidy committed:
willpower seperated uni19, taros - added nonmem for manslayeronly

wtffidy committed:
Merge branch 'Skua' of https://github.com/BrenoHenrike/Scripts into Skua

wtffidy committed:
Skill Combo Updates: Requested by; Rafocale#0629

updates:
Archpaladin
Immortal Dark Caster
Frost SpiritReaver
HighSeas Commander
Nechronomancer

LordExelot committed:
AutoEnhance: Prepped for Forge Enhancements p2

wtffidy committed:
Release: FreeBoosts(10mns) :fried_shrimp:

--can either run the script or use the other void inside to use the void elsewhere

wtffidy committed:
Create ThreeSpellStory.cs

wtffidy committed:
maybe thill fix NDW

wtffidy committed:
maybe now

LordExelot committed:
LightCaster: changed string name to int id for checkinventory

LordExelot committed:
Core: Added off option to end map

LordExelot committed:
Adv: Fixed buyAllMerge and added better logging to CanBuy

LordExelot committed:
Merge branch 'Skua' of https://github.com/BrenoHenrike/Scripts into Skua

LordExelot committed:
Released: AshfallCampMerge

LordExelot committed:
MergeTempalteHelper: Now does BankingBlackList

ShaunH321 committed:
Formatting

tanigorlx committed:
Classtype from farm to solo since boss

tanigorlx committed:
efficiency

LordExelot committed:
Archfiend: Advanced BuyItem

LordExelot committed:
Merge branch 'Skua' of https://github.com/BrenoHenrike/Scripts into Skua

wtffidy committed:
Updated Fishing Rep :thumbsup:

wtffidy committed:
bait quant 50 > 20 (dcs happen)

wtffidy committed:
preped for release

LordExelot committed:
Core: Increased stability on shop loading

LordExelot committed:
Merge branch 'Skua' of https://github.com/BrenoHenrike/Scripts into Skua

wtffidy committed:
why!?

LordExelot committed:
Various: Moved a bunch of CBO Other settings to Script Options

LordExelot committed:
Merge branch 'Skua' of https://github.com/BrenoHenrike/Scripts into Skua

wtffidy committed:
:thumbsup:

hopefully i didnt break anything im ill be at work

ShaunH321 committed:
Create ArmyVoucherItemTotemofNulgath.cs

wtffidy committed:
rest is filled in :thumbsup:

wtffidy committed:
put a heart of the sun bool for the new enh

hopefully this works?

wtffidy committed:
uni35 redo

uni35 is a random reward form the quest so :+1:  added an if it gets it bit or if it gets 9 arch ess before it gets the random drop, to just buy it

wtffidy committed:
would bugout without the update(prolly)

wtffidy committed:
Merge branch 'Skua' of https://github.com/BrenoHenrike/Scripts into Skua

wtffidy committed:
Update UnlockForgeEnhancements.cs

wtffidy committed:
Update UnlockForgeEnhancements.cs

wtffidy committed:
Update UnlockForgeEnhancements.cs

PUNK3DAF committed:
UnlockForgeEnhancements: rouge lmao

PUNK3DAF committed:
Merge branch 'Skua' of https://github.com/BrenoHenrike/Scripts into Skua

PUNK3DAF committed:
UnlockForgeEnhancements: please push ffs

PUNK3DAF committed:
CoreYnR: Wrong item ID and typo

wtffidy committed:
redoing this because something fekd up

wtffidy committed:
recreating this, git got fuckedup somehow

--made by Shaun

wtffidy committed:
redoing this--

wtffidy committed:
redoing - git messup

PUNK3DAF committed:
UnlockForgeEnhancements: cs_include fix

wtffidy committed:
redo finished?

wtffidy committed:
updatequest needed?

wtffidy committed:
Merge branch 'Skua' of https://github.com/BrenoHenrike/Scripts into Skua

wtffidy committed:
Update CoreAdvanced.cs

wtffidy committed:
Release: DageRecruitMerge

Requester: @Zepiroz#9789

wtffidy committed:
ReleaseNavalTopHatMerge

Requester: @D-A+R=2#1920

wtffidy committed:
Update DarkWarLegionandNation.cs

wtffidy committed:
Release: ShadowSlayerK

Requester: Phoenix#5869

PUNK3DAF committed:
UnlockForgeEnhancements: Cape quests fixed a bit

ShaunH321 committed:
Update ShadowSlayerK.cs

ShaunH321 committed:
Update ShadowSlayerK.cs

PapaShokry committed:
added a space before ArtixWeddingQuests

tanigorlx committed:
put ensure accept and complete for quest 8829

LordExelot committed:
Adv: Enabled AutoEnhance update

LordExelot committed:
Documentation: Made start to add Skua and Core info

PUNK3DAF committed:
ShadowSlayerK: BuyQuest fix

PUNK3DAF committed:
Merge branch 'Skua' of https://github.com/BrenoHenrike/Scripts into Skua

PUNK3DAF committed:
ShadowSlayerK: Drop added

PUNK3DAF committed:
ShadowSlayerK: Typo and EnsureAccept missing

PUNK3DAF committed:
ShadowSlayerK: i dont even know anymore, just bugs

PUNK3DAF committed:
ShadowSlayerK: Basically reworked the whole script

wtffidy committed:
Update AdvancedSkills.txt

wtffidy committed:
update combos

wtffidy committed:
isTemp: false

PUNK3DAF committed:
ShadowslayerSummoningRitual added

PUNK3DAF committed:
Merge branch 'Skua' of https://github.com/BrenoHenrike/Scripts into Skua

PUNK3DAF committed:
ShadowslayerSummoningRitual: Storyline added

wtffidy committed:
Update ShadowslayerSummoningRitual.cs

PUNK3DAF committed:
Merge branch 'Skua' of https://github.com/BrenoHenrike/Scripts into Skua

wtffidy committed:
made it a foreach :smirk:  and added the missing  ShadowStory.Storyline();

wtffidy committed:
Update ShadowslayerSummoningRitual.cs

wtffidy committed:
added updatequest required for book wyrn, and someone forgor the istemp:false for eternity blade

wtffidy committed:
Update ShadowslayerSummoningRitual.cs

wtffidy committed:
Update ShadowslayerSummoningRitual.cs

wtffidy committed:
seperated ShadowSlayersApprentice and put it before the rest in story & quest item farm

wtffidy committed:
Update ShadowSlayerK.cs

LordExelot committed:
Multiple Cores: Streamlined files and prepped for Docs

LordExelot committed:
Documentation: 7/9 Docs completed

wtffidy committed:
hazmap fixed, Artixpointe renovated to the newer style :thumbsup:

wtffidy committed:
Merge branch 'Skua' of https://github.com/BrenoHenrike/Scripts into Skua

ShaunH321 committed:
Free slots stopsbot now - like it should.

wtffidy committed:
updated to new standard.

PUNK3DAF committed:
DageRecruit: Mob fix, DarkWarLegionandNation: Map fix

wtffidy committed:
Released

Requester: @vanetti01#1208

wtffidy committed:
Merge branch 'Skua' of https://github.com/BrenoHenrike/Scripts into Skua

wtffidy committed:
optimistic > timeout:250

wtffidy committed:
void loadstone was to accept the quest not for nothinacess :facepunch:

wtffidy committed:
potions update to make it use less gold... hopefully it works

wtffidy committed:
Update UltraPotions.cs

wtffidy committed:
Update UltraPotions.cs

PUNK3DAF committed:
TrickstersGiftsMerge: tato

wtffidy committed:
isTemp > falsle.. im derpin today

wtffidy committed:
added a use locked maps bool

wtffidy committed:
fogor quanatities

wtffidy committed:
Update DarkWarLegionandNation.cs

LordExelot committed:
Farm: Fixed inverted bool

wtffidy committed:
Merge branch 'Skua' of https://github.com/BrenoHenrike/Scripts into Skua

LordExelot committed:
FollowerJoe: Removed old bit

wtffidy committed:
Merge branch 'Skua' of https://github.com/BrenoHenrike/Scripts into Skua

LordExelot committed:
Core: Rearanged functions to make more sense for the docs

LordExelot committed:
Documentation/Core/CoreBots: 9/9

LordExelot committed:
Core: Changed message

LordExelot committed:
Docs: Added ToDo List

LordExelot committed:
Docs: Removed redundant </a>

LordExelot committed:
Docs: Fixed links

PUNK3DAF committed:
UnlockForgeEnhancements: a teeny tiny fix for Penitence

PUNK3DAF committed:
Merge branch 'Skua' of https://github.com/BrenoHenrike/Scripts into Skua

tanigorlx committed:
updated cs includes

tanigorlx committed:
update quest id

wtffidy committed:
lament - quants fixed

wtffidy committed:
Merge branch 'Skua' of https://github.com/BrenoHenrike/Scripts into Skua

wtffidy committed:
Update ThreeSpellStory.cs

wtffidy committed:
Update AlchemyREP.cs

wtffidy committed:
Update UnlockForgeEnhancements.cs

wtffidy committed:
Update AlchemyREP.cs

LordExelot committed:
Create CoreStory.md

LordExelot committed:
Docs: Released CoreStory

LordExelot committed:
Docs: Underline on todo

LordExelot committed:
Docs: Test

PapaShokry committed:
Update BeleensMerge.cs

LordExelot committed:
Docs: Test^2

LordExelot committed:
Docs: Test

LordExelot committed:
Docs/Story: Formatting

LordExelot committed:
Docs/Story: Test

LordExelot committed:
Docs/Story: Formatting test

LordExelot committed:
Docs/Story: Formatting test

LordExelot committed:
Docs/Story: Formatting v20341273

LordExelot committed:
Docs/Story: Rewrote table for formatting

LordExelot committed:
Docs/Story: Coloring

LordExelot committed:
Docs/Story: Missing =

LordExelot committed:
Docs/Story: test

LordExelot committed:
Docs/Story: Color test

LordExelot committed:
Docs/Story: test

LordExelot committed:
Width test

LordExelot committed:
Shadow Slayer: Fix for Odokuro

LordExelot committed:
Adv: Better AutoEnhance Logger

LordExelot committed:
Core: Removed crash from logger

LordExelot committed:
MysteriousEgg: Added logger if you have manticore

LordExelot committed:
Docs/Story: test

LordExelot committed:
Core:  Fixed potential crash of registerquest

ShaunH321 committed:
fixed not unbanking before farm.

wtffidy committed:
small changes mostly to logging & areas with more then 1 of the hunted mobs

wtffidy committed:
Merge branch 'Skua' of https://github.com/BrenoHenrike/Scripts into Skua

wtffidy committed:
Temp fix for Tokena

i mean the issue is with buyitem  buying TokenA whilst either

A. the quant it wants to get (anything more then 1) it doesnt ave the mats for and wont continue

B.it has both 200c and 20b, and tries to buy 1A, which it has and stops

wtffidy committed:
_ break shit? Temp Fix rever w/e ya fix the _ stuff

ShaunH321 committed:
UpdateQuest added or it will get stuck.

ShaunH321 committed:
Merge branch 'Skua' of https://github.com/BrenoHenrike/Scripts into Skua

tanigorlx committed:
Add files via upload

tanigorlx committed:
fix format

PapaShokry committed:
fixing for forgeenhancment "heart of the sun" item required

 reported by :@Genysis#9030

PapaShokry committed:
Merge branch 'Skua' of https://github.com/PapaShokry/Scripts into Skua
